### PR TITLE
Clean up grammar in Microsoft SSO guide

### PR DESCRIPTION
**Pull request by carolstran**
🧹 Since we're about to add new SSO guides, I wanted to clean up the language in the existing Microsoft SSO guide.

This was only a super quick sweep. Future TODOs: Updating the language and image alt texts to be more accessible ✨ 

Related internal ticket 🎟️  [CI-1771](https://emnify.atlassian.net/browse/CI-1771)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1567,25 +1567,25 @@ With Single Sign-On enabled members of your organization with access to the EMni
 * An Azure Subscription with an Active Directory license
 * An account in the EMnify portal using the same email address as the one used to sign in to Microsoft Azure
 
-WARNING: If your email address used in the EMnify portal is not the same as the one used to login to Azure, this setup will not work. Instead, add an additional user to the EMnify portal with `Administrator` privileges. 
+WARNING: If your email address in the EMnify portal differs from the one used to log in to Azure, this setup won't work. Instead, add an additional user to the EMnify portal with `Administrator` privileges. 
 
 ==== SSO with Microsoft
 
-Log into Azure and navigate to Azure Active Directoring in the left sidebar.
+Log into Azure and navigate to `Azure Active Directory` in the left sidebar.
```
Comment (techwriter-dev) [inline README.adoc:1574]:
```suggestion
Log in to Azure and navigate to `Azure Active Directory` in the left sidebar.
```

File: README.adoc [hunk 2]
```diff
@@ -1594,7 +1594,7 @@ image:sso_api_permissions_s6.png[set api permissions]
 Your configuration should look like this:
 image:sso_api_permissions_s7.png[api permission configuration]
 
-In the left sidebar, go to *Expose an API* and click on `Set` and then click on `save`. 
+In the left sidebar, go to *Expose an API*, click on `Set`, and then click on `save`.
```
Comment (techwriter-dev) [inline README.adoc:1597]: For the sake of consistency:
```suggestion
In the left sidebar, go to *Expose an API*, click `Set`, and then click `save`. 
```

File: README.adoc [hunk 3]
```diff
@@ -1608,29 +1608,30 @@ image:sso_copy_application_id_s12.png[copy application id]
...
+Fill in the *Client ID* and the *Client Secret* with the values you copied earlier, then click `Create and Activate`. 
 image:sso_create_activate_s16.png[create and activate]
 
-The final step is to verify the provider. SSO will not be configured until you complete this step. Click on *Verify Integration* and follow the prompts.
+The final step is to verify the provider. You must complete this step to configure SSO. Click *Verify Integration* and follow the prompts.
 image:sso_verify_s17.png[verify the sso provider]
 
 ==== Troubleshooting
-In the unlikely event that the provider could not be configured on the very first attempt, check the following:
+If a member of your organization runs into issues while setting up SSO with Microsoft Active Directory, check the following:
```
Comment (techwriter-dev) [inline README.adoc:1627]:
```suggestion
If your organization encounters issues while setting up SSO with Microsoft Active Directory, check the following:
```

File: README.adoc [hunk 4]
```diff
@@ -1608,29 +1608,30 @@ image:sso_copy_application_id_s12.png[copy application id]
...
 
-* If the user receives a ‘User cancelled social login' check if the application registration is set to Multi-tenant
-* Is the user part of the Azure Tenant
-* Is the user part of the EMnify enterprise
-* Is the user enabled
-* Try again using incognito tab
+* Do they see a ‘User canceled social login' message? If yes, make sure the application registration is set to Multitenant.
```
Comment (techwriter-dev) [inline README.adoc:1629]: For the sake of consistency (see line 1581).
```suggestion
* Do they see a ‘User canceled social login' message? If yes, make sure the application registration is set to *Multitenant*.
```

**Review by techwriter-dev — COMMENTED** [review]
Added suggestions mainly to ensure formatting and word choice consistency.

File: README.adoc (hunk 3 above)
Comment (carolstran) [reply README.adoc:1627]: This was settled in the other PR so not adding it to the batch

File: README.adoc (hunk 4 above)
Comment (carolstran) [reply README.adoc:1629]: Does this still apply now that this troubleshooting section is generic and added to the end of the SSO section? 🤔

Comment (carolstran) [reply README.adoc]: Changed it regardless, but let me know and I can update for the PR to the new docs platform.

Comment (techwriter-dev) [reply README.adoc]: @carolstran Let's discuss this in real time when you are available. I'm approving for now.

**Review by techwriter-dev — APPROVED** [review]
Ready to merge.

**Pull request merged**